pkg/cli/cmd.go

```diff
@@ -142,15 +145,41 @@ type MetricsBuilder interface {
 	Build() (io.Reader, error)
 }
 
-func (t Telemetry) CustomMetricsHandlerFor(mb MetricsBuilder) http.Handler {
+func (t Telemetry) CustomMetricsHandlerFor(reg *prometheus.Registry, mb MetricsBuilder) http.Handler {
```

Comment:
Passing `Gatherer` instead of `Registry`.

Reply:
Nice catch 👍 Address by https://github.com/pipe-cd/pipe/pull/2279/commits/6513a2d87a265e2d656c8514a66d3ee2d8782886 🙏